var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -40,7 +43,7 @@ class Crtm(CMakePackage):
     # depends_on("ecbuild", when="@2.4.0:", type=("build"))
 
     # REL-2.4.0_emc (v2.4.0 ecbuild does not work)
-    version("2.4.0", commit="5ddd0d6")
+    version("2.4.0", commit="5ddd0d6")  # use commit="690da37" for JCSDA-internal
```

Comment:
Would it be helpful to add a commented version statement with the JCSDA-internal commit hash? (Like the url's above).

Reply:
Ok, can do.

Reply:
Done, please have a look